.github/ISSUE_TEMPLATE/1-bug.yml

```
@@ -0,0 +1,86 @@
+name: 🐛 Bug report
+description: Describe a problem
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some general tips:
+
+        - Is this really a problem?
+        - Is this a problem here?
+        - Can this be solved in a different way?
+
+        Relevant links:
+
+        [Support Docs](https://github.com/syntax-tree/.github/blob/main/support.md)
+        [Contributing Guide](https://github.com/syntax-tree/.github/blob/main/contributing.md)
+        [Code of Conduct](https://github.com/syntax-tree/.github/blob/main/code-of-conduct.md)
+        [Starter query for searching existing issues](https://github.com/search?q=user%3Asyntax-tree&type=Issues)
+
+  - type: checkboxes
+    id: initial-checklist
+    attributes:
+      label: Initial checklist
+      options:
+        - label: I read the support docs
+          required: true
+        - label: I read the contributing guide
+          required: true
+        - label: I agree to follow the code of conduct
+          required: true
+        - label: I searched issues and couldn’t find anything (or linked relevant results below)
+          required: true
+
+  - type: input
+    id: affected-versions
+    attributes:
+      label: Affected packages and versions
+      description: Please test using the latest version of the relevant packages to make sure your issue has not already been fixed. Also make sure you’re on recent versions of Node and npm.
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        How did this happen?
+        Please provide a minimal, reproducible example:
+        https://stackoverflow.com/help/minimal-reproducible-example
+        Issues without reproduction steps or code examples may be immediately closed
+        as not actionable.
+      value: |
+        1.
+        2.
+        3.
+        ...
+    validations:
+      required: true
+
+  - type: input
+    id: sandbox
+    attributes:
+      label: Sandboxed Example
```

```suggestion
      label: Link to sandbox (optional)
```

- The casing of the label is currently different than the rest (which use sentence case).
- I think sandbox is a bit jargon, especially “sandboxed”, is there an alternative that is less related to CodeSandbox, but also not so vague like “Runnable code” where folks will just paste a link to their website or complete project repo?
- As this is the only optional field and an alternative version of the previous textarea, I think it’s good to highlight that it’s optional. Also possible: `Link to sandbox (if applicable)`

---

reframed this as "runnable example" and added some commentary, let me know what you think of the new version.